pkg/client/clientset.go

```diff
@@ -248,7 +248,7 @@ func CreateClientSet(ctx context.Context, url string, auth manifest.Auth, opts C
 		WithRetryOptions(&DefaultRetryOptions).
 		WithRateLimiter(true)
 
-	if opts.SupportArchive {
+	if v := ctx.Value(support.SupportArchive{}); v != nil && v.(*support.SupportArchive).Value {
```

Comment:
This line is different to the one in `dynatrace.go`; in any case I'd like to see the check put in a function in the `support` package

Reply:
done